sm_python/python/sm/__init__.py

```diff
@@ -83,3 +83,18 @@ def logFatalNamed(name,message):
 
 def logErrorNamed(name,message):
     logNamed(name, LoggingLevel.Error, message, stackdepth=2)
+
+def setExtendableValueStoreValue(evs, key, value, add_if_missing=True):
+    type2func_string = {
+        bool : "Bool",
+        int : "Int",
+        float : "Double",
+        str : "String"
+    }
```

Comment:
Consider making this a global var (constant).

Reply:
I changed this to a function doing if on isinstance, as that one is able to detect np.float64 propperly where this is not. wdyt? would you still make the function public/global?

Reply:
It is fine to keep it as a local function. Sad, that the mapping doesn't work as well.